extensions/EXT/EGL_EXT_explicit_device.txt

```diff
@@ -0,0 +1,101 @@
+Name
+
+    EXT_explicit_device
+
+Name Strings
+
+    EGL_EXT_explicit_device
+
+Contributors
+
+    Adam Jackson
+    Nicolai Haehnle
+    Daniel Stone
+
+Contacts
+
+    Adam Jackson <ajax@redhat.com>
+
+Status
+
+    Complete
+
+Version
+
+    Version 2, 2017-08-08
+
+Number
+
+    EGL Extension #122
+
+Extension Type
+
+    EGL device extension
+
+Dependencies
+
+    Requires EGL_EXT_platform_base and EGL_EXT_device_base.
+
+    EGL_EXT_platform_device trivially interacts with this extension.
+
+    This extension is written against the EGL 1.5 Specification.
+
+Overview
+
+    A system may support multiple devices and multiple window systems. For
+    example, a Wayland environment may drive multiple GPUs and support both
+    X11 and EGL clients. In order to realize this, the implementation must
+    allow the client to specify both device and platform explicitly.
+
+    The EGL_EXT_platform_device extension enables the client to create a
+    display from a device, but does so by overloading the native_display
+    argument to eglGetPlatformDisplay; the user passes the device as the
+    native display, and presumably the EGL picks a sensible default for the
```

Comment:
This is not an accurate description of EGL_EXT_platform_device.  To start, the terms "platform" and "window system" are meant to be synonymous.  As such, platform device explicitly replaces the need for a window system because it is itself a EGL "platform" or window system option for EGL, and it's not overloading the native_display argument any more than any other EGL platform would.